### ISSUE TITLE
hide "some people can reply" posts 

### ISSUE THREAD

**Issue by davimack** [bluesky-social/social-app#7110]
### Describe the Feature

Users should be able to hide "some people can reply" posts from their feed. These are essentially the same as advertisements. Graying out the comment icon isn't the same, as users may only notice the gray after reading the post, when trying to comment.

### Describe Alternatives

Gray the entire title, maybe? Still not the same. If the poster wants to be hidden, I should be able to honor that.

**Comment by surfdude29** [comment]
You may not be able to reply to those posts, but you might still be able to comment on them by quoting the post (although the author may have disabled that as well). So any setting might want to take that into consideration too.

**Comment by mlynch** [comment]
Would love to see this. Feels very wrong that I'm forced to see their post but not allowed to reply. At least make it configurable

**Comment by bikerglen** [comment]
Yes! Please add an option to hide posts / reposts that I can't reply too!

**Comment by ldldjr** [comment]
Why should I waste my time reading the comments of people who don't want to see my well thought out replies?
